shardingsphere-kernel/shardingsphere-transaction/shardingsphere-transaction-core/src/main/java/org/apache/shardingsphere/transaction/core/TransactionType.java

```diff
@@ -22,7 +22,7 @@
  */
 public enum TransactionType {
     
-    LOCAL, XA, BASE;
+    LOCAL, XA, BASE, MANUALXA;
```

Comment:
I cannot agree to add new transaction type here.
I don't think `MANUALXA` is a valid transaction type, it is not a same abstract level with `LOCAL`, `XA` and `BASE`

Reply:
`MANUALXA` is not a good name. It might be better to use `NONE` or `TRANSPARENT`. you know, naming is a hard work!